addons/website/static/src/js/website.ace.js

```diff
@@ -367,6 +488,32 @@ var ViewEditor = Widget.extend({
         window.location.hash = "";
         this.$el.removeClass('oe_ace_open').addClass('oe_ace_closed');
     },
+    check_customized:function(){
+        if(this.editor_type != 'html' && this.selectedViewId() && (this.selectedViewId().lastIndexOf('/custom/', 0) === 0)){
+            $('.custom_ace_view').removeClass('hidden');
+        }
+        else{
+            $('.custom_ace_view').addClass('hidden');
+        }
```

Comment:
May be use toggleClass (it's shorter)
var view = this.selectedViewId();
var is_custom = this.editor_type != 'html' && view && view.lastIndexOf('/custom/', 0) === 0;
$('.custom_ace_view').toggleClass('hidden', is_custom);